src/tests/ftest/performance/performance_test_base.py

```diff
@@ -0,0 +1,401 @@
+#!/usr/bin/env python3
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+import re
+import time
+import types
+
+from ior_test_base import IorTestBase
+from mdtest_test_base import MdtestBase
+from mdtest_utils import MdtestMetrics
```

Comment:
(pylint-no-name-in-module) No name 'MdtestMetrics' in module 'mdtest_utils'
(pylint-unused-import) Unused MdtestMetrics imported from mdtest_utils